interactive.js

```diff
@@ -131,6 +162,48 @@ const handleFkillError = async processes => {
 	}
 };
 
+const DEFAULT_EXIT_TIMEOUT = 3000;
+
+const performKillSequence = async processes => {
+	if (!Array.isArray(processes)) {
+		processes = [processes];
+	}
+
+	let survived;
+	let hadError;
+	try {
+		await fkill(processes);
+		const exited = await Promise.all(processes.map(process => processExited(process, DEFAULT_EXIT_TIMEOUT)));
+		survived = processes.filter((_, i) => !exited[i]);
+	} catch (error) {
+		survived = processes;
+		hadError = error;
+	}
+
+	if (survived.length > 0) {
```

Comment:
Use early-return.

Reply:
You sure about that? Using early return for normal way of things and putting rare case handling in the main flow seems weird and misleading to me.